REQUIREMENTS.md

```
@@ -1,289 +1,240 @@
 
-オフセット検出：onset 強度の相互相関最大点 → ±10ms 以内を目標。
+OGP：/og.png（ヒーローのUIサンプル＋簡潔コピー）
 
-De-Esser：i≈6（素材により ±2 dB）
+14. パフォーマンス
 
-Compressor：threshold -18dB, ratio 2:1, attack 15ms, release 120ms
+LCP ≤ 2.5s（3G相当でもFVテキストが即読める）
 
-ダッキング：threshold -20dB, ratio 2:1, mix 0.7
+画像は<img loading="lazy">／音源はpreload="none"
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)